packages/driver/src/cy/multi-domain/util.ts

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash'
+import { AssertionError } from 'chai'
+
+export const correctStackForCrossDomainError = (serializedError: any, userInvocationStack: string) => {
+  //  Since Errors sent over postMessage need to be serialized to Objects, we need to serialize them back into Error instances
+  let errorClass
+
+  switch (serializedError?.name) {
+    case 'AssertionError':
+      errorClass = AssertionError
+      break
+    default:
+      errorClass = Error
+  }
+
+  // only assign missing properties on the object to reify the error properly
+  const reifiedError = _.assignWith(new errorClass(serializedError?.message), serializedError, (objValue, srcValue) => {
+    return _.isUndefined(objValue) ? srcValue : objValue
+  })
+
+  reifiedError.stack = userInvocationStack
```

Comment:
```suggestion
  reifiedError.stack = $stackUtils.replacedStack(reifiedError, userInvocationStack)
```

In chrome-like browsers, `error.stack` includes `error.message`, so straight up replacing it will cause a mismatch between `error.message` and the message that's included in `error.stack`. This method replaces the stack while keeping the error's message.

Reply:
oh cool! Should be fixed now and the errors printing out to the console now contain the error.